src/utils/userLocation.ts

```diff
@@ -59,46 +54,16 @@ export const updateCurrentLocationData = ({
   userId,
   profileLocationData,
 }: UpdateCurrentLocationDataProps) => {
-  const [locationName] = Object.keys(profileLocationData);
-
   updateLocationData({
     userId,
-    newLocationData: { [locationName]: getCurrentTimeInMilliseconds() },
+    newLocationPath: profileLocationData,
   });
 };
 
 // TODO: refactor how user location updates works here?
 //   Called from VenuePage useEffect + onBeforeUnloadHandler
 export const clearLocationData = (userId: string) => {
-  updateLocationData({ userId, newLocationData: {} });
-};
-
-export interface EnterExternalRoomProps {
-  userId: string;
-  roomUrl: string;
-  locationName: string;
-}
-
-export const enterExternalRoom = ({
-  userId,
-  locationName,
-  roomUrl,
-}: EnterExternalRoomProps) => {
-  setLocationData({
-    userId,
-    locationName,
-  });
-
-  logEventGoogleAnalytics({
```

Comment:
Is anybody looking at these events?

Reply:
No, it was done for some hacky way of tracking users and has never been removed